zanata-war/src/main/webapp-jboss/WEB-INF/web.xml

```diff
@@ -157,6 +157,11 @@
   <!--</listener>-->
 
   <filter-mapping>
+    <filter-name>CharacterEncodingFilter</filter-name>
```

Comment:
It would be good to have a comment, explaining why this filter needs to be first: otherwise the character set might accidentally be locked to ISO-8859-1.